Rakefile

```diff
@@ -43,8 +43,7 @@ namespace :build do
 
   desc "Builds a tarball."
   task :tar => [:version_files, :build_file, :precompile_assets] do
-    include_file = Build::Productization.file_for("config/tarball/include")
```

Comment:
I guess this means you get to also delete the include file?  :scissors:  :+1: Please also delete that file if it's not needed.